src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerWaitContainer.groovy

```diff
@@ -16,9 +16,17 @@
 package com.bmuschko.gradle.docker.tasks.container
 
 class DockerWaitContainer extends DockerExistingContainer {
+
+    Integer exitCode;
+
+    DockerWaitContainer() {
+        ext.getExitCode = { exitCode }
+    }
+
     @Override
     void runRemoteCommand(dockerClient) {
         logger.quiet "Waiting for container with ID '${getContainerId()}'."
-        dockerClient.waitContainerCmd(getContainerId()).exec()
+        exitCode = dockerClient.waitContainerCmd(getContainerId()).exec()
```

Comment:
I think it's fair to assume that we want to fail the build if the exit code != 0. I don't think we necessarily need to expose the exit code. What's the use case?